### ISSUE TITLE
[BUG] - Disabled Input Fields

### ISSUE THREAD

**Issue by BeezyIV** [vultr/whmcs-vultr#51]
**Describe the bug**
OS Type & Auto backups Dropdown Fields Disabled in both Admin & Clientarea.

**To Reproduce**
1. Go to 'My Services' from your WHMCS client area
2. Click on 'Click on your Active VPS service'
3. Scroll down to down

**Expected behavior**
1. The "OS TYPE" drop-down field should show a list of Operating systems available thus allowing the client to choose their preferred operating system.

2. The "AUTO BACKUPS" field should show the options "YES/NO" when clicked on providing the user to choose whether to enable or disable auto backups.
 
**Screenshots**
Screenshot of the issue in the client area - ![VM Setup Client Area](https://user-images.githubusercontent.com/34535237/68126786-3764d400-ff1d-11e9-97eb-42497d44313a.JPG)
Screenshot of the affected fields in admin back-end - ![VM Setup Admin Area](https://user-images.githubusercontent.com/34535237/68126788-3764d400-ff1d-11e9-8962-952fab2cf600.JPG)

**Desktop (please complete the following information where applicable:**
 - CentOS 7.7
 - PHP7.3, MySQL 5.6 & 5.7
 - Google Chrome & Mozilla Firefox
 - WHMCS Version 7.8.3
- Vultr WHMCS module v2.0.4

**Additional context**
I also fount this piece of code in public_html/whmcs/modules/servers/vultr/template/controller/main/create.tpl . Could this have anything to do with my issue?
See screenshot.

![Code snippet of vultrOSID](https://user-images.githubusercontent.com/34535237/68127096-d4c00800-ff1d-11e9-83f0-663f872a13e3.JPG)

I can provide login details to my fresh install of WHMCS admin dashboard if reproduction of the issue fails on your end.

**Comment by ddymko** [comment]
@BeezyIV 

For the OS type did you generate your default values for these?

https://github.com/vultr/whmcs-vultr/issues/21#issuecomment-509215907
https://github.com/vultr/whmcs-vultr/issues/25#issuecomment-515057865

`Setup -> Addon Modules -> Vultr Module -> Configure`

As for the autobacks this may be we will look into this if it is not resolved by the generating your default values

**Comment by BeezyIV** [comment]
@ddymko 
Yes I had the default values generated. But by creating a new product and generating the default values,  I've had the OS and application drop down to work. Thanks. However, the OS Type drop down is only enabled in the admin back-end and on the order form. It still remains disabled in the client area.
1. Is this how it is supposed to be?
2. When a client successfully deploys a server, the server info shown is from vultr. Is it supposed to be like this or is there a way to change this? 
3. Backup field is still disabled and leaving selecting NO on order form doesn't change anything.

Once again thanks for your response. Please refer to the following screenshots for further details.

![OS Type on Order form](https://user-images.githubusercontent.com/34535237/68144020-625e2080-ff3b-11e9-99e9-b07f496de77f.JPG)

![Client area fields](https://user-images.githubusercontent.com/34535237/68144021-62f6b700-ff3b-11e9-8c99-8c65ee9356a0.JPG)

![Admin backend fields](https://user-images.githubusercontent.com/34535237/68144022-638f4d80-ff3b-11e9-8740-6a1adbd69cf9.JPG)

**Comment by dezzyboy** [comment]
hey guys im having this same issue and it looks like there is some error with the module 
<img width="1222" alt="Screen Shot 2020-03-07 at 2 45 13 PM" src="https://user-images.githubusercontent.com/37994033/76151374-54074900-6082-11ea-87fa-33c9051a7e34.png">

**Comment by dezzyboy** [comment]
> hey guys im having this same issue and it looks like there is some error with the module
> <img alt="Screen Shot 2020-03-07 at 2 45 13 PM" width="1222" src="https://user-images.githubusercontent.com/37994033/76151374-54074900-6082-11ea-87fa-33c9051a7e34.png">

Correction
so the error above is irrelevant , however there are a few steps that needs to be done in order to get this module setup and os type and application to populate.. i will add a screen shot for a step by step guide to help anyone else that faces this issue

**Comment by SaneChoice** [comment]
Hi All,

For the OS Type please see my post below. I had to do a few things to make the values show in the order process and also the client configuration. 

https://github.com/vultr/whmcs-vultr/issues/77
